app/api/server/v1/push.js

```diff
@@ -63,3 +66,16 @@ API.v1.addRoute('push.token', { authRequired: true }, {
 		return API.v1.success();
 	},
 });
+
+API.v1.addRoute('push.get', { authRequired: true }, {
+	get() {
+		const params = this.requestParams();
+		check(params, Match.ObjectIncluding({
+			id: String,
+		}));
+
+		const data = PushNotification.getNotificationForMessageId(params.id, Meteor.userId());
```

Comment:
need to add a validation if `Meteor.userId()` can access the room of the given message `id`